src/NetAnalyzers/Core/AbstractGlobalizationDiagnosticAnalyzer.cs

```diff
@@ -0,0 +1,28 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using Analyzer.Utilities;
+using Microsoft.CodeAnalysis.Diagnostics;
+
+namespace Microsoft.CodeAnalysis.NetAnalyzers
+{
+    public abstract class AbstractGlobalizationDiagnosticAnalyzer : DiagnosticAnalyzer
+    {
+        protected virtual GeneratedCodeAnalysisFlags GeneratedCodeAnalysisFlags { get; } = GeneratedCodeAnalysisFlags.None;
```

Comment:
Should I remove this and always use `None`?

Reply:
i think we can keep it for future use